lib/modules/tooltip.js

```diff
@@ -294,15 +294,11 @@ export class _TooltipTarget extends React.Component {
   }
 
   render() {
-    const { inline, type, children } = this.props;
+    const { children: child } = this.props;
 
-    if (typeof children === 'array') {
-      throw new Error('Tooltip target can only have one child');
-    } else if (typeof children === 'string') {
-      return <span {...this.makeHandler()}>{ children }</span>;
-    } else {
-      return React.cloneElement(children, this.makeHandler());
-    }
+    return typeof child === 'string'
+      ? <span { ...this.makeHandler() }>{ child }</span>
+      : React.cloneElement(React.Children.only(child), this.makeHandler());
```

Comment:
Woo, it's everywhere 🎉  probably want to add children to  propTypes for _TooltipTarget as well